options/locale/locale_en-US.json

```diff
@@ -3781,10 +3781,10 @@
   },
   "git.filemode": {
     "changed_filemode": "%[1]s → %[2]s",
-    "directory": "Directory",
-    "normal_file": "Normal file",
-    "executable_file": "Executable file",
-    "symbolic_link": "Symbolic link",
-    "submodule": "Submodule"
+    "directory": "directory",
+    "normal_file": "normal",
```

Comment:
I think we should hide the "normal" prompt, almost every file is "normal".

It's really strange to keep seeing a lot of "normal" on the UI: what indeed is "normal" if end users don't know it refers to the "file mode"?